1주차 과제.css

```
@@ -0,0 +1,159 @@
+body{
+    margin:0;
+    padding:0;
+}
+.banner{
+    background-color: #4B89DC;
+    width:100%;
+    height:64px;
+    border: none;
+    margin-top:0px;
+    
+}
+#begin_lo{
+    width:117px;
+    height:28px;
+    font-size: 20px;
+    color:white;
+    margin-top: 0px 0px 0px 0px;
```

```suggestion
    margin-top: 0px;
```
이렇게 수정했을 때 차이가 있을까요?